README.md

```diff
@@ -80,4 +89,7 @@ java -jar plugin-management-cli/target/plugin-management-tool.jar -p "workflow-s
 
 #### Other Information
 The plugin manager tries to use update center data to get the latest information about a plugin's dependencies. If this information is unavailable, it will use the dependency information from the downloaded plugin's MANIFEST.MF file.
+
 For plugins listed in a .txt file, each plugin must be listed on a new line. Comments beginning with `#` will be filtered out.
+
+Support for downloading plugins from maven is not yet supported, but is a feature that is planned to be added in the future [JENKINS-58217](https://issues.jenkins-ci.org/browse/JENKINS-58217)
```

Comment:
not sure if this line is needed, have users been asking for it?
seems like `url` covers most needs if they aren't in an update centre

Reply:
It was brought up once on the channel and once during the last GSoC evaluation presentation.